cmd/ir2proxy/main.go

```diff
@@ -38,31 +38,36 @@ func main() {
 		log.Fatal(err)
 	}
 
-	contourscheme.AddToScheme(scheme.Scheme)
-
+	// Split the yaml file by docs, and parse each one individually
 	for _, yamldoc := range bytes.Split(data, []byte("---")) {
```

Comment:
top tip: if you make a function call "splitYaml" then you probably don't need the comment.